frontend/src/lib/components/PropertyIcon.tsx

```diff
@@ -51,6 +51,7 @@ export const PROPERTIES_ICON_MAP = {
         ['desktop']: <IconMonitor />,
         ['mobile']: <IconPhone />,
         ['tablet']: <IconTablet />,
+        ['console']: <IconHeadset />,
```

Comment:
Should we have a wearable one here as well? Even if it's just a placeholder for now

Reply:
Sounds good to keep it consistent, just added it.